javamelody-core/src/main/java/net/bull/javamelody/internal/model/MBeansAccessor.java

```diff
@@ -120,7 +119,7 @@ static double getDoubleFromOperatingSystem(String attribute) {
 	}
 
 	static long getThreadAllocatedBytes(long threadId) {
-		if (!MBEAN_ALLOCATED_BYTES_ENABLED) {
+		if (!isMbeanAllocatedBytesEnabled()) {
```

Comment:
This method may be called a lot. This change may cause a hotspot and a regression of performance, so we should probably reverse this line and change the constant instead.

Reply:
```suggestion
		if (!MBEAN_ALLOCATED_BYTES_ENABLED) {
```